CLI crash both on windows 2012 server and ubuntu 14.04

### Package Versions

Replace the values below using the output from `npm list storj`.

C:\Users\Administrator
└── (empty)

npm ERR! code 1

C:\Users\Administrator>

I'm getting this output from windows and ubuntu. 


Replace the values below using the output from `node --version`.

v4.4.5


### Actual Behavior

The program crashes at some stage:
{"level":"warn","message":"dropping received late response to 38afa306ee37d1e80816d95054a5730ee4c88644","timestamp":"2016-06-28T22:47:37.235Z"}
{"type":"info","message":"sent telemetry report \"Success\"","timestamp":"2016-06-28T22:47:37.235Z"}
C:\Users\Administrator\AppData\Roaming\npm\node_modules\storjshare-cli\node_modules\storj\lib\network\protocol.js:199
    item.trees[contract.get('renter_id')] = params.audit_tree;
                       ^

TypeError: Cannot read property 'get' of undefined
    at C:\Users\Administrator\AppData\Roaming\npm\node_modules\storjshare-cli\node_modules\storj\lib\network\protocol.js:199:24
    at C:\Users\Administrator\AppData\Roaming\npm\node_modules\storjshare-cli\node_modules\storj\lib\manager.js:71:5
    at C:\Users\Administrator\AppData\Roaming\npm\node_modules\storjshare-cli\node_modules\storj\lib\storage\adapters\level\index.js:55:7
    at C:\Users\Administrator\AppData\Roaming\npm\node_modules\storjshare-cli\node_modules\storj\lib\storage\adapters\level\filestore.js:25:5
    at dispatchError (C:\Users\Administrator\AppData\Roaming\npm\node_modules\storjshare-cli\node_modules\storj\node_modules\levelup\lib\util.js
    at C:\Users\Administrator\AppData\Roaming\npm\node_modules\storjshare-cli\node_modules\storj\node_modules\levelup\lib\levelup.js:222:14

C:\Users\Administrator>


### Steps to Reproduce

Just leave it working for a while. It can crash in a few minutes or hours.

--- a/lib/network/protocol.js
+++ b/lib/network/protocol.js
@@ -196,6 +196,11 @@ Protocol.prototype._handleConsign = function(params, callback) {
 
     var contract = item.contracts[params.contact.nodeID];
     var t = Date.now();
+
+    if (!contract) {
+      return callback(new Error('Consignment is not authorized'));
+    }
+
     item.trees[contract.get('renter_id')] = params.audit_tree;
 
     try {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "storj",
-  "version": "1.3.3",
+  "version": "1.3.4",
   "description": "implementation of the storj protocol for node.js and the browser",
   "main": "index.js",
   "directories": {

--- a/test/network/protocol.unit.js
+++ b/test/network/protocol.unit.js
@@ -7,6 +7,7 @@ var Logger = require('kad').Logger;
 var KeyPair = require('../../lib/keypair');
 var stream = require('readable-stream');
 var constants = require('../../lib/constants');
+var StorageItem = require('../../lib/storage/item');
 
 describe('Protocol', function() {
 
@@ -230,6 +231,23 @@ describe('Protocol', function() {
         contact: { nodeID: 'adc83b19e793491b1c6ea0fd8b46cd9f32e592fc' }
       }, function(err) {
         expect(err.message).to.equal('Failed');
+        done();
+      });
+    });
+
+    it('should error if the contract is for different nodeid', function(done) {
+      var proto = new Protocol({
+        network: {
+          _logger: Logger(0),
+          _manager: {
+            load: sinon.stub().callsArgWith(1, null, StorageItem({}))
+          }
+        }
+      });
+      proto._handleConsign({
+        contact: { nodeID: 'adc83b19e793491b1c6ea0fd8b46cd9f32e592fc' }
+      }, function(err) {
+        expect(err.message).to.equal('Consignment is not authorized');
         done();
       });
     });

